src/olympia/addons/views.py

```diff
@@ -153,7 +153,10 @@ def persona_detail(request, addon):
     if not (addon.is_public() or addon.is_pending()):
         raise http.Http404
 
-    persona = addon.persona
+    try:
+        persona = addon.persona
```

Comment:
Add a small comment why this happens? Although it's only very temporary till legacy pages are thrown away…

Reply:
Yeah, the API workaround is clearly documented because it's here to stay, but the legacy pages... meh.